tests/pipeline/test_merge_blood.py

```diff
@@ -155,9 +155,18 @@ def test_merge_blood(spark_session):
 
     df_voyager_bloods, df_bloods_residuals, df_failed_records = merge_blood(voyager_df, bloods_df)
 
+    # expected_df_voyager_bloods.toPandas().to_csv("expected_df_voyager_bloods.csv", index=False)
```

Comment:
Please remove these if not used. Also please check for any other debugging (`pdb`) statements